<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>531</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="taskNames">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Названия задач</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reportButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Сгенерировать отчет</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>План работ</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Импорт данных</string>
     </property>
     <addaction name="import_plan_2"/>
     <addaction name="import_hierarchy"/>
    </widget>
    <addaction name="plan_group"/>
    <addaction name="plan"/>
    <addaction name="menu_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Фактическое выполнение</string>
    </property>
    <addaction name="fact"/>
    <addaction name="import_fact"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Общая информация</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="plan_group">
   <property name="text">
    <string>Добавить группу</string>
   </property>
  </action>
  <action name="plan">
   <property name="text">
    <string>Добавить задачу</string>
   </property>
  </action>
  <action name="fact">
   <property name="text">
    <string>Данныые о задаче</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О проекте</string>
   </property>
  </action>
  <action name="import_fact">
   <property name="text">
    <string>Импорт данных</string>
   </property>
  </action>
  <action name="import_plan_2">
   <property name="text">
    <string>План работ</string>
   </property>
  </action>
  <action name="import_hierarchy">
   <property name="text">
    <string>Иерархия</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
